cmd/auto-pause/auto-pause.go

```diff
@@ -103,6 +103,7 @@ func runPause() {
 	mu.Lock()
 	defer mu.Unlock()
 	if runtimePaused {
+		out.Step(style.AddonEnable, "Auto-pause is already enabled.")
```

Comment:
this doesnt need to be a Step,
Step is specially used for Progress bar

Reply:
Yes, we can use "out.Styled" instead of "out.Step".

https://pkg.go.dev/k8s.io/minikube/pkg/minikube/out#Styled

At the end of this function (line 124) there is a call to "out.Step". Should this be replaced with "out.Styled" for consistency?

Reply:
I believe that one should stay, as line 124 is saying "The containers have paused" and that's essentially a step.